### PR TITLE
Minor undertow tests fixes, HTTPSListenerTestCase now runs in domain …

### PR DESCRIPTION
**Pull request by honza-kasik**
…too. This should reduce number of failing tests in undertow.

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java [hunk 1]
```diff
@@ -101,6 +101,11 @@ public String createHTTPSListener(String httpServer) throws IOException, Command
         String name = RandomStringUtils.randomAlphanumeric(6);
         String securityRealm = "SecurityRealm_ssl-test";
         Address realmAddress = Address.coreService("management").and("security-realm", securityRealm);
+        if (client.options().isDomain) {
+            realmAddress = Address.host(client.options().defaultHost)
+                .and("core-service", "management")
+                .and("security-realm", securityRealm);
+        }
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java]: Looking at the Creaper code, this should be handled automatically even for core-services, see `org.wildfly.extras.creaper.core.online.AdjustOperationForDomain#adjust(org.jboss.dmr.ModelNode)`

BTW: Any reason for not using prepared command in creaper `org.wildfly.extras.creaper.commands.undertow.AddHttpsSecurityRealm`?

**Comment by honza-kasik** [comment]
Now using command instead of batch to add security realm.

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java [hunk 2]
```diff
@@ -20,10 +21,10 @@
 import org.wildfly.extras.creaper.core.CommandFailedException;
 import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
 import org.wildfly.extras.creaper.core.online.operations.Address;
-import org.wildfly.extras.creaper.core.online.operations.Batch;
+//import org.wildfly.extras.creaper.core.online.operations.Batch;
 import org.wildfly.extras.creaper.core.online.operations.OperationException;
 import org.wildfly.extras.creaper.core.online.operations.Operations;
-import org.wildfly.extras.creaper.core.online.operations.Values;
+//import org.wildfly.extras.creaper.core.online.operations.Values;
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/UndertowOperations.java]: Please remove the commented imports.

**Comment by rhatlapa** [comment]
Please remove the commented imports and squash, I will then merge it.

**Comment by honza-kasik** [comment]
Import removed and squashed...

**Comment by rhatlapa** [comment]
Thanks, merging

**Pull request merged**